### PR TITLE
Update to newest simplesamlphp

### PR DESCRIPTION
**Pull request by legionth**
Updates the simplesamlephp package to the newest version 1.15.3.

References:
https://packagist.org/packages/simplesamlphp/simplesamlphp

### CODE REVIEW

**Comment by legionth** [comment]
Ping @mjansenDatabay

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Please add the relevant information in the "extra" section.

**Comment by legionth** [comment]
Added the "information" additionally in the extra section. Have a look.

File: libs/composer/composer.json [hunk 1]
```diff
@@ -233,7 +233,7 @@
 		},
 		"simplesamlphp/simplesamlphp" : {
 			"source": "https://github.com/simplesamlphp/simplesamlphp",
-			"used_version": "v1.14",
+			"used_version": "v1.15.3",
 			"added_by": "Michael Jansen <mjansen@databay.de>",
 			"wrapped_by": null,
 			"last_update": "2017-07-12",
```
Comment (mjansenDatabay) [inline libs/composer/composer.json]: Please update the "last_update" property.

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]

**Comment by legionth** [comment]
Solved changes. Have a look @mjansenDatabay

File: libs/composer/composer.json [hunk 2]
```diff
@@ -233,10 +233,10 @@
 		},
 		"simplesamlphp/simplesamlphp" : {
 			"source": "https://github.com/simplesamlphp/simplesamlphp",
-			"used_version": "v1.14",
+			"used_version": "v1.15.3",
 			"added_by": "Michael Jansen <mjansen@databay.de>",
 			"wrapped_by": null,
-			"last_update": "2017-07-12",
+			"last_update": "2018-03-01",
 			"last_update_by": "Michael Jansen <mjansen@databay.de>",
```
Comment (mjansenDatabay) [inline libs/composer/composer.json]: "last_update_by" is also missing :-)

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]

File: libs/composer/composer.json (hunk 2 above)
Comment (legionth) [reply libs/composer/composer.json]: See newest commit

**Comment by legionth** [comment]
Updated the PR and rebased it. The update of the dependencies although updated the development dependencies. I rebased the branch to avoid these updates.

**Comment by mjansenDatabay** [comment]
@legionth Can you please update your branch to the current 5.3.x revision, then update the lib. to v. 1.15.4, and finally adopt the changes I made for the files in `./Services/Saml/`in the trunk?

* https://github.com/ILIAS-eLearning/ILIAS/commit/40a6b6b7a15cf939e0f1ad73a82e7215873591c2
* https://github.com/ILIAS-eLearning/ILIAS/commit/03e5d1425284cb16e358eb55999e8a01bd608bbe

Many thanks in advance.

**Comment by legionth** [comment]
Hey @mjansenDatabay,

updated to recommended simplesaml version. Also cherry-picked the mentioned commits.

Happy "Review" :sweat_smile:

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**